EIPS/eip-5719.md

```diff
@@ -0,0 +1,87 @@
+
+---
+eip: 5719
+title: Signature replacement interface
+description: Non-interactive replacing of smart contract wallet signatures that became stale due to configuration changes.
+author: Agustin Aguilar (@Agusx1211)
+discussions-to: https://ethereum-magicians.org/t/erc-signature-replacing-for-smart-contract-wallets/11059
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
If this proposal is modifying the RPC interface wallets expose, it should be in the `Interface` category.

Reply:
It doesn't affect the RPC interface.